pages/account/[[...index]].tsx

```diff
@@ -196,29 +194,52 @@ export async function getServerSideProps({ req }) {
     }
   }
   // Parsing path from url to pass to ProfileTabs.
-  const tabsPath = req.url.split("/", -1)[2] || null
+  const tabsPath = req.url.split("/").slice(2).join("/") || null
```

Comment:
I wonder if a regex would make this less brittle and demonstrate more clearly what we're extracting from the url.

Reply:
```
const tabsPathRegex = /\/(?:[^\/]+\/){2}(.+)/
const match = req.url.match(tabsPathRegex)
const tabsPath = match ? match[1] : null
```
is something like this what you were thinking? to me it's kind of not clearer

Reply:
i guess I was imagining something with some part of the url explicitly in it like
`/\/account\/(.+)\/*/`
my thinking is seeing the capture group next to what we expect the url to be is clearer. i also don't imagine this would happen, but this regex is a little more flexible for any kind of domain change since it's reliant on the /account part of the string. as i see it, that's the only thing that's guaranteed in the url for this page. but also, that's not a serious concern. 
if you think it's still not clearer, its fine to leave as is. i just thought it was worth exploring.